src/droplet.F

```diff
@@ -394,13 +401,13 @@ subroutine droplet_driver(dt,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,    &
           iflag = 1
         ELSE
           ! cm1r19:
-          call find_horizontal_location_index (pdata_locind(np,1), x3d, ib, ie+1, xf, ni+2, iflag)
+          call find_horizontal_location_index (pdata_locind(np,1), pdata(np,prx), ib, ie+1, xf, ni+2, iflag)
```

Comment:
Could @drichte2 explain why using `pdata(np,prx)`, `pdata(np,pry)` and `pdata(np,prz)` instead of using `x3d`, `y3d` and `z3d` here?

Reply:
This is related to the jflag issue -- at this point, these are identical so it technically doesn't matter which one you use. I switched to "pdata" because these contain all the new values. i.e., rk2_integration and BE_integration end with applying the parcel boundary conditions and then setting pdata(np,prx)=x3d, pdata(np,pry)=y3d, pdata(np,prz)=z3d.

Reply:
Thanks @drichte2 . That makes sense to me.